package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lune",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Can you remove this for now. I will bump the version in a separate commit when I publish to npm. It makes it easier to track when versions are bumped.